server/controllers/users.js

```diff
@@ -0,0 +1,198 @@
+import jwt from 'jsonwebtoken';
+import db from '../models';
+import Helpers from './helpers';
+
+/**
+ * Secret token for jsonwebtoken
+ */
+const secret = process.env.SECRET || 'my secret key';
+
+const usersController = {
+  /**
+   * Login a user
+   * @param {Object} req - Request object
+   * @param {Object} res - Response object
+   * @returns {Object} Response object
+   */
+  login(req, res) {
+    db.User.findOne({ where: { email: req.body.email } })
+      .then((user) => {
+        if (user) {
+          if (user.validatePassword(req.body.password)) {
+            const token = jwt.sign({ userId: user.id, roleId: user.roleId}, 
+            secret, { expiresIn: '1 day' });
+            return res.status(200).send({token, user});
+          }
+        }
+        res.status(401)
+            .send({ message: 'Invalid Login Details!' });
+      });
+  },
+
+ /**
+   * Create a user
+   * @param {Object} req - Request object
+   * @param {Object} res - Response object
+   * @returns {Object} Response object
+   */
+  create(req, res) {
+    db.User.findOne({ where: { email: req.body.email } })
+      .then((existingUser) => {
+        if (existingUser) {
+          return res.status(400).send({ 
+            message: `Oops! A user already exists with this email: ${req.body.email}` 
```

Comment:
Don't return the email. A more generic message like `email is taken` will be better.